src/tests/ftest/util/dmg_utils.py

```diff
@@ -130,6 +131,15 @@ def hostlist(self, hostlist):
                 hostlist = ",".join(hostlist)
             self._hostlist.update(hostlist, "dmg._hostlist")
 
+    def set_certificates(self, hostlist):
+        """Set certificates for the hostlist.
+
+        Args:
+            hostlist (string list): list of host addresses
```

Comment:
The '()' should contain a python type.
```suggestion
            hostlist (list): list of host addresses
```